Esri/arcgis-maps-sdk-swift-samples
# [New] Identify KML features

## Description

This PR implements `Identify KML features` in `Layers` category.
URL to README: [README](https://github.com/Esri/arcgis-maps-sdk-swift-samples/tree/Caleb/New-IdentifyKMLFeatures/Shared/Samples/Identify%20KML%20features)

## Linked Issue(s)

- `swift/issues/4599`

## How To Test

- Tap on a feature to view the KML placemark information in a callout.
- Note: If "No Significant Weather Is Expected" is showing, change the `forecastKML` link in the `URL` extension at the bottom of the file to another [NOAA KML file link](https://www.wpc.ncep.noaa.gov/kml/noaa_chart/) with data, such as https://www.wpc.ncep.noaa.gov/kml/noaa_chart/WPC_Day1_SigWx_2022071012.kml.


## Screenshots

![identify-kml-features](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/34120440/0d38fdd2-233c-4289-b181-dc2bee32894d)

## To Discuss

- Note: The `NSAppTransportSecurity` key had to be added to the `Info.plist` in order to get the KML layer's legend to load. This is due to the link to the image being `http` instead of `https`.
